internal/schema/usage_data.go

```diff
@@ -149,3 +152,50 @@ func ParseAttributes(i interface{}) map[string]gjson.Result {
 
 	return a
 }
+
+func MergeAttributes(dst *UsageData, src *UsageData) {
+	for key, srcAttr := range src.Attributes {
+		if _, has := dst.Attributes[key]; has {
+			switch srcAttr.Type {
+			case gjson.Null:
+				fallthrough
+			case gjson.True:
+				fallthrough
+			case gjson.False:
+				fallthrough
+			case gjson.Number:
+				fallthrough
+			case gjson.String:
+				// Should be safe to override
+				dst.Attributes[key] = srcAttr
+			case gjson.JSON:
+				var err error
+				var destJson map[string]interface{}
+				var srcJson map[string]interface{}
+				err = json.Unmarshal([]byte(dst.Attributes[key].Raw), &destJson)
+				if err != nil {
+					log.Error(err)
```

Comment:
```suggestion
					log.Errorf("Error merging attribute '%s': %v", key, err)
```
I think for these log lines we should add some additional context, like above, so it helps us debug if there is any issue.

Reply:
Done